BananaBread/bb.html

```diff
@@ -1203,6 +1203,9 @@
       <progress value="0" max="100" id="progress" hidden=1></progress>
     </div>
 
+    <br>
```

Comment:
I think you're thinking of XHTML, which is probably from around 2000, yeah. In HTML you [shouldn't close the tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br).